vizro-core/docs/pages/tutorials/first-dashboard.md

```diff
@@ -2,7 +2,7 @@
 
 There is no setup needed for your first dashboard, thanks to the amazing [Py.Cafe](https://py.cafe/).
 
-Click on the **Run and edit this code in Py.Cafe** link below to live-edit the dashboard.
+Click on the **Run and edit this code in py.cafe** link below to live-edit the dashboard.
```

Comment:
This is such an amazing change. I must be a genius

Reply:
Do you have insight onto the style guide for use of "py.cafe" vs "Py.Cafe" that I don't? I have noticed they're calling it [`Py.cafe` here](https://py.cafe/terms) but [`py.cafe` in the ToS](https://app.termly.io/policy-viewer/policy.html?policyUUID=6a4b19ee-881e-4940-867c-3bc978e5ec84) and [`PyCafe` on the About page](https://py.cafe/about). We've consistently used `Py.Cafe` throughout and I think that is definitely wrong.

Maybe a GHC ticket to correct it to "PyCafe" where we are talking about the organisation and "py.cafe" when talking specifically about "navigating to py.cafe" that is, the website.

Reply:
thanks, as revealed on Slack, this PR was from me (Max) :P

Reply:
Good question, we prefer to use PyCafe, or the link https://py.cafe

Reply:
Thanks, i'll note that and create an issue to fix anything else.